keyboards/basekeys/trifecta/keyboard.json

```diff
@@ -102,7 +108,89 @@
             "multisplash": true,
             "solid_splash": true,
             "solid_multisplash": true
-        }
+        },
```

Comment:
Needs `"driver": "ws2812"` above; RGB Matrix is not enabled by default so CI didn't catch this.